tubular/nominal.py

```diff
@@ -549,7 +549,7 @@ def __init__(
         weights_column: str | None = None,
         prior: int = 0,
         level: str | list | None = None,
-        unseen_level_handling: str | int | float | None = None,
+        unseen_level_handling: str | float | None = None,
```

Comment:
N - is this just because int is redundant or is this a fix for something?

Reply:
It's redundant.

From [PEP 484](https://peps.python.org/pep-0484/#the-numeric-tower):

> Rather than requiring that users write import numbers and then use numbers.Float etc., this PEP proposes a straightforward shortcut that is almost as effective: when an argument is annotated as having type float, an argument of type int is acceptable; similar, for an argument annotated as having type complex, arguments of type float or int are acceptable. 

Good chat in [this thread](https://stackoverflow.com/questions/50928592/mypy-type-hint-unionfloat-int-is-there-a-number-type).

Reply:
Whilst this PR has been approved - I'll just wait a day or so to check that something unexpected has happened here. We aren't in a rush.

Reply:
I had a double check of the code path and am confident that we'll be ok. I'll merge this in.